docs/references/general/grid.md

```diff
@@ -0,0 +1,103 @@
+<!-- Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan 
+author(s): Marina Schmidt 
+
+Disclaimer: License under GNU v3.0, the file is found in the root directory under LICENSE 
+
+-->
+# GRID files 
+
+GRID files are post-processed data produced from FITACF files.
+
+## Naming Conventions
+
+### Single radar GRID files
+
+Currently the common naming convention for a single radar GRID file is:
+
+> YYYYMMDD.HH.mm.ss.<3-letter abbreviation>.grid
+
+SuperDARN radars routinely change operating frequency. Some operating modes use multiple frequencies, either at the same time, or by alternating between frequencies. As a result, some files from some radars include all records for all frequencies in the same file and other radars separate out the individual frequencies into separate files, designated by a channel letter in the file name:
+
+> YYYYMMDD.HH.mm.ss.<3-letter abbreviation>.[a-d].grid
+
+See [fitacf](fitacf.md) documentation for more information on this naming convention. 
+
+If a GRID file is produced for 24-hours using the `-c` option in `make_grid` (see [`make_grid` tutorial](../../user_guide/make_grid.md)) then the common naming convention is:
+
+> YYYYYMMDD.<3-letter abbreviation>.grid
+
+### Combined GRID files 
+
+After GRID files are produced for separate radars, they are sometimes further combined with other radars GRID files to make a GRD file. This file has the same format as a GRID file, but includes data from multiple radars. 
+See [combine_grid](../../user_guide/make_grid.md) on how to make GRD files. 
+
+The common naming convention for 24-hr GRD files is:
+
+> YYYYMMDD.grd  
```

Comment:
> @aburrell: I also recommend specifying that separate files are created for the northern and southern hemispheres, and by default all available radars in a given hemisphere are included.

I've tried to address the first part of this here. I think the second part belongs in the [make_grid](docs/user_guide/make_grid.md) documentation

Side note: I'm starting to think that the GRID/GRD distinction is more confusing than helpful. 

```suggestion
### Combined "GRD" files for each hemisphere

After GRID files are produced for separate radars, they may then be combined with other radars' GRID files to make a GRD file. This file has the same format as a GRID file, but includes data from multiple radars. 
See [combine_grid](../../user_guide/make_grid.md) on how to make GRD files. 

Combined GRD files are produced for the northern and southern hemispheres separately and usually contain 24 hours of data. The common naming convention for 24-hr GRD files is:

> YYYYMMDD.north.grd  
> YYYYMMDD.south.grd  
```